tasks/vulcanize.js

```diff
@@ -11,6 +11,8 @@
 module.exports = function(grunt) {
 
   var vulcanize = require('vulcanize');
+  var fileSystem = require('fs');
```

Comment:
Is this necessary? Looks like `node-fs` does not implement the functions used below, but they are a part of the normal `fs` module.